prometheus_hardware_exporter/collectors/redfish.py

```diff
@@ -252,14 +266,21 @@ def get_storage_controller_data(self) -> Tuple[Dict[str, int], Dict[str, List]]:
         for system_id in system_ids:
             storage_controller_count[system_id] = 0
 
+            # finding storage name used in URI by querying Systems dictionary
```

Comment:
Can you explain in the comment why we want to find storage name instead of hard code?

Reply:
Good point, I'll mention about how the name can sometimes be non-standard and we need to query it dynamically.